static/templates/admin_user_group_list.hbs

```diff
@@ -9,9 +9,6 @@
             <img class="checkmark" src="/static/images/checkbox-green.svg" />
             {{t 'Saved' }}
         </button>
-        <button class="button save-status btn-danger small">
-            {{t 'Discard changes' }}
-        </button>
         <button class="button rounded small delete btn-danger">
```

Comment:
Can you explain in the commit message why we are doing this? Link to the issue if there is any open.

Reply:
I think 'Discard' is used to undo the last changes (Ex: for adding the user), but there is no need to do so because we can already change the group members by one click on 'x' (on the right of the person name). That's why i make a decision to eliminate the 'Discard' button, which looks to me more UI upgraded.

Reply:
https://chat.zulip.org/#narrow/stream/9-issues/topic/.2316465 discussion link

Reply:
The "Discard" button is there to undo changes that have been made but not submitted to the server -- we use this pattern in all of "organization settings".

Reply:
I don't think it makes sense to remove the "Discard changes" button given that it provides useful functionality (E.g. in a very large group when you remove someone by accident with a misclick)

Reply:
@timabbott ok. should i remove the discard button commit and make a PR with "delete button" change with "delete icon" only.